rdkit/ML/Data/DataUtils.py

```diff
@@ -154,15 +155,15 @@ def ReadQuantExamples(inFile):
 
   """
   expr1 = re.compile(r'^#')
-  expr2 = re.compile(r'[\ ]*|[\t]*')
```

Comment:
Python 3 complains about a possibly empty regex. Changing the * to + makes sure that we split on at least one whitespace.